managed/src/main/java/com/yugabyte/yw/controllers/TablesController.java

```diff
@@ -97,17 +98,8 @@ public Result create(UUID customerUUID, UUID universeUUID) {
       resultNode.put("taskUUID", taskUUID.toString());
       auditService().createAuditEntry(ctx(), request(), Json.toJson(formData.data()));
       return Results.status(OK, resultNode);
-    } catch (NullPointerException e) {
-      LOG.error("Error creating table", e);
-      // This error isn't useful at all, why send a NullPointerException as api response?
-      return ApiResponse.error(BAD_REQUEST, "NullPointerException");
-    } catch (RuntimeException e) {
```

Comment:
Where is RuntimeEx coming from?

Reply:
I think this is not required.  Just removed try-catch block